### ISSUE TITLE
Re-search problem

### ISSUE THREAD

**Issue by Michahel** [savanto/AnDICT-Android#12]
On the screen there is constantly displaying for a lengly time the message "Looking up word…" and the word search doesn't find any definition which has the desired expression.
 
**How to reproduce the problem**
 
Go to the AnDICT Settings. Set up options:
```
DICT server address: dict.bible.ru
DICT server port: 2628
Databases to search: All matches
Use search strategy: enable
Search strategy: word
```
In the "Enter a word to look up" box type the word "исус" and tap on Lookup.
The word "исус" contains a mistake. Such a word is not in any of the dictionary base. In that situation the message "Looking up word…" pops up on the screen until you tap somewhere outside of that message.
Tap on the word on the "Enter a word to look up" box. Fix the mistake and type the word "Иисус". Tap on Lookup. On the screen the message "Looking up word…" still pops up for a long time, as if there is no such word in the dictionary base. Now you cannot find even one word until you restart the program.

**Comment by Michahel** [comment]
I decided to come up with another search query to help you locate the problem. First type in "esus" instead of "исус" and then search for "Jesus". You will get the same effect.

**Comment by savanto** [comment]
@Michahel This should be fixed in [v1.3.3](https://github.com/savanto/AnDICT-Android/releases/tag/v1.3.3). Please upgrade and test when you have a chance, and let me know if you find any other problems.

**Comment by Michahel** [comment]
It works correctly. I checked.
